src/cpp/session/http/SessionHttpConnectionUtils.cpp

```diff
@@ -212,6 +197,10 @@ bool checkForSuspend(boost::shared_ptr<HttpConnection> ptrConnection)
       }
       else
       {
+         if (http_methods::connectionDebugEnabled())
+         {
+            LOG_DEBUG_MESSAGE("handle session suspend");
```

Comment:
Does this (and the below stubs) need to be implemented before merging this PR?

Reply:
Those are just part of a debug logging feature, so you'll see "handle session interrupt" or "handle save_document" in the logs one for each request received. I recognize it reads kind of funny in the code now that you mention it :)